schemas/post-artifact-request.yml

```diff
@@ -31,11 +31,35 @@ oneOf:
            always using `application/json` for JSON artifacts.
         type:       string
         maxLength:  255
+      contentMD5:
+        description: |
+          Base64 encoded MD5 sum of the artifact. If specified, it **must**
+          also be given as value to the `Content-MD5` header when uploading
+          to S3.
+          **Warning**, this will likely be required in the future, and **must**
+          be specified if `contentSHA256` is specified.
+        type:       string
```

Comment:
This applies to the entire PR, but adding it here. Is there a reason we prefer to base64 decode the MD5/SHA256 rather than leave them in their _already ascii_ form?